### PR TITLE
Fix TS Repair weapons versus values

### PR DESCRIPTION
**Pull request by abcdefg30**
[Rules.ini](https://github.com/LipkeGu/TSIniFiles/blob/master/rules.ini#L8583)

### CODE REVIEW

**Comment by GraionDilach** [comment]
We shouldn't even have `Versus` defined on that one tbh, just give it a `TargetType: Repair` á lá RA1 mechanic.

**Comment by pchote** [comment]
I agree with @GraionDilach.  This is just propagating the broken legacy "0% means untargetable" logic

**Comment by abcdefg30** [comment]
Updated.

File: mods/ts/weapons/healweapons.yaml [hunk 1]
```diff
@@ -17,17 +17,19 @@ Repair:
...
 	Projectile: Bullet
 		Speed: 1c682
 	Warhead@1Dam: SpreadDamage
 		Spread: 213
 		Damage: -50
+		ValidTargets: Repair
 		Versus:
 			None: 0
-			Wood: 0
+			Wood: 100
```
Comment (GraionDilach) [inline mods/ts/weapons/healweapons.yaml]: Remove Versus altogether instead of refining it. :D

**Comment by abcdefg30** [comment]
Updated.

**Comment by GraionDilach** [comment]
:+1:

**Pull request merged**